### ISSUE TITLE
Changing account settings without retyping of password

### ISSUE THREAD

**Issue by benkees** [nextcloud/mail#2600]
### Feature Request

After changing an account's settings, retyping of the password should not be necessary. Resetting the password, should be an extra step/menu.

**Comment by ChristophWurst** [comment]
is this a bug report?

**Comment by benkees** [comment]
Hi Christoph,
I wouldn't consider it a bug. Do you need any other information?

**Comment by ChristophWurst** [comment]
No, now I actually just got what this is about :) This is a UX bug at most, thus feature request makes sense :+1:

Technically speaking we shouldn't send the foll account details for an update, but just the props that changed. This would allow changing account details without typing in the password.

**Comment by benkees** [comment]
Exactly! :)
